bin/test_travis.sh

```diff
@@ -227,6 +234,15 @@ EOF
     unset USE_SYMENGINE
 fi
 
+if [[ "${TEST_OPT_DEPENDENCY}" == *"pycosat"* ]]; then
```

Comment:
This is already done by the above https://github.com/sympy/sympy/pull/17440/commits/5688c89865208bb503a5a29e9604a91e19bcbe32#diff-97e2e3c68308a6044377595c72c26cedR199

Reply:
I guess you removed those lines. Having it in those lists is the correct way, with a single test/doctest run for all the modules that use optional dependencies.

Reply:
Ok, done